app/Controller/PagesController.php

```diff
@@ -74,3 +120,44 @@ public function display() {
 		}
 	}
 }
+
+/**
+ * CyclomaticComplexity
+ */
+// Cyclomatic Complexity = 12
+class Foo {
```

Comment:
__[staging][code_sniffer]__
* Each class must be in a file by itself
* Each class must be in a namespace of at least one level (a top-level vendor name)